Content.Server/Power/Components/CablePlacerComponent.cs

```diff
@@ -12,5 +12,8 @@ public sealed partial class CablePlacerComponent : Component
 
         [DataField("blockingWireType")]
         public CableType BlockingCableType = CableType.HighVoltage;
+
+        [DataField]
+        public bool OverTile = false;
```

Comment:
If you're adding fields please describe what they do.

Reply:
Done!